packages/proto-codecs/src/codegen/osmosis/cosmwasmpool/v1beta1/genesis.ts

```diff
@@ -97,3 +153,81 @@ export const GenesisState = {
     };
   },
 };
+export const PoolI_InterfaceDecoder = (
```

Comment:
This section looks good, but one thing I notice is that there isn't really any error handling. Given what we have seen in the sentry, is it worth wrapping in an error handler?

Reply:
I think errors here would be unexpected